python/PyQt6/core/__init__.py.in

```diff
@@ -92,11 +94,6 @@ QgsVectorLayer.NoMarker = Qgis.VertexMarkerType.NoMarker
 QgsVectorLayer.NoMarker.is_monkey_patched = True
 QgsVectorLayer.NoMarker.__doc__ = "No marker"
 
-QgsSymbol.RenderHints = Qgis.SymbolRenderHints
```

Comment:
Why are these removed? Are they duplicates?

Reply:
They are generated by the new sipify, for instance, you got these for `Qgis.SymbolRenderHints`

```
Qgis.SymbolRenderHints = lambda flags=0: Qgis.SymbolRenderHint(flags)
QgsSymbol.RenderHints = Qgis.SymbolRenderHints
```